tests/accessor/accessor_api_common_buffer_local.h

```diff
@@ -77,7 +77,7 @@ template <typename T, cl::sycl::access::mode mode,
           cl::sycl::access::target target,
           cl::sycl::access::placeholder placeholder>
 void multidim_subscript_write(
-    cl::sycl::accessor<T, 1, mode, target, placeholder>& acc,
+    const cl::sycl::accessor<T, 1, mode, target, placeholder>& acc,
```

Comment:
Just double checking that you want `const` when writing into `acc`.
I think this is right, since I had to use some `mutable` in the triSYCL implementation.

Reply:
Yes, we still need it to be const and accessor operator[] that are used here is const according to SYCL-2020 spec p. 160